tests/phpunit/tests/post/nav-menu.php

```diff
@@ -550,22 +550,22 @@ function test_wp_nav_menu_filters_are_passed_args_object() {
 	/**
 	 * Run tests required to confrim Walker_Nav_Menu receives an $args object.
 	 */
-	function _confirm_nav_menu_item_args_object( $args ) {
+	public function confirm_nav_menu_item_args_object( $args ) {
 		$this->assertIsObject( $args );
 		return $args;
 	}
 
-	function _confirm_second_param_args_object( $ignored_1, $args ) {
+	public function confirm_second_param_args_object( $ignored_1, $args ) {
 		$this->assertIsObject( $args );
 		return $ignored_1;
 	}
 
-	function _confirm_third_param_args_object( $ignored_1, $ignored_2, $args ) {
+	public function confirm_third_param_args_object( $ignored_1, $ignored_2, $args ) {
 		$this->assertIsObject( $args );
 		return $ignored_1;
 	}
 
-	function _confirm_forth_param_args_object( $ignored_1, $ignored_2, $ignored_3, $args ) {
+	public function _confirm_forth_param_args_object( $ignored_1, $ignored_2, $ignored_3, $args ) {
```

Comment:
@costdev Why does this one remain prefixed with `_` while the others have been changed?

Reply:
🙈 What underscore?

_Corrected. Force pushing momentarily..._